redesign/community.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
```

Comment:
I hadn't noticed before but we're missing the `lang` attribute - useful for screen readers. 
https://www.w3.org/International/questions/qa-html-language-declarations#attributes